modules/datadog/main.tf

```diff
@@ -1,24 +1,4 @@
-resource "kubernetes_secret" "datadog_keys" {
-  metadata {
-    name      = "datadog-keys"
-    namespace = var.namespace
-  }
-
-  data = {
-    api-key = datadog_api_key.datadog_agent.key
-    app-key = datadog_application_key.datadog_agent.key
-  }
-
-  depends_on = [module.datadog_operator]
-}
 # Datadog Operator
-resource "datadog_api_key" "datadog_agent" {
-  name = coalesce(var.datadog.agent_api_key_name, var.cluster_name)
-}
-
-resource "datadog_application_key" "datadog_agent" {
-  name = coalesce(var.datadog.agent_app_key_name, var.cluster_name)
-}
 
 locals {
   datadog_site = "datadoghq.eu"
```

Comment:
here too, if it's a module we should have this as an input.

Reply:
This part has been removed, since we no longer create the keys.

Reply:
This code has been removed since no longer used.